reloj_checador_team03/test/controllers/tmp/rubycritic/assets/javascripts/application.js

```diff
@@ -0,0 +1,281 @@
+prettyPrint();
```

Comment:
HERE BE DRAGONS 🐉 

There is no need on tracking these`tmp` files

You can always run Rubycritic in your CI pipeline (it could be GitHub Actions)